app/subscriber/src/features/manage-folder/ManageFolder.tsx

```diff
@@ -0,0 +1,171 @@
+import { Checkbox, Col, IContentModel, IFolderModel, Row } from 'tno-core';
+
+import * as styled from './styled';
+
+/**
+ * Component that renders the manage folder page, allowing user to view/manage content inside a folder
+ * @returns ManageFolder component
+ */
+export const ManageFolder: React.FC = () => {
+  const { id } = useParams();
+  const [, { getFolder, updateFolder }] = useFolders();
+  const navigate = useNavigate();
+  const [, { findContent }] = useContent();
+  const [folder, setFolder] = React.useState<IFolderModel>();
+  const [items, setItems] = React.useState<any>([]);
+  const [selected, setSelected] = React.useState<IContentModel[]>([]);
+
+  /** TODO: Folder content only contains contentId and sortOrder so we have to make an additional call based off of the contentIds to get the headline/summary etc..
+   * assuming we want this to differ eventually.
+   */
+  React.useEffect(() => {
+    getFolder(Number(id)).then((folder) => {
+      // have to use temp not folder state as folder state will not update right away
+      const temp = folder;
+      setFolder(folder);
+      findContent({
+        contentIds: folder.content.map((c) => c.contentId),
+      })
+        .then((data) => {
+          const tempSort = data.items.map((item) => ({
+            ...item,
+            sortOrder: temp.content.find((c) => c.contentId === item.id)?.sortOrder ?? 0,
+          }));
+          tempSort.sort((a, b) => a.sortOrder - b.sortOrder);
+          setItems(tempSort);
+        })
+        .catch();
+    });
+    // Only on initialize, or when sort order changes
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  /** function that runs after a user drops an item in the list */
+  const handleDrop = React.useCallback(
+    async (droppedItem: any) => {
+      if (!droppedItem.destination) {
+        return;
+      }
+      var updatedList = [...items];
+      // hold response
+      let res: IFolderModel | undefined;
+      // Remove dragged item
+      const [reorderedItem] = updatedList.splice(droppedItem.source.index, 1);
+      // Add dropped item
+      updatedList.splice(droppedItem.destination.index, 0, reorderedItem);
+      // Update State
+      setItems(updatedList);
+      // Update Folder
+      if (!!folder) {
+        res = await updateFolder({
+          ...folder,
+          content: updatedList.map((item, index) => ({
+            ...item,
+            contentId: item.id,
+            sortOrder: index,
+          })),
+        });
+      }
+      setFolder(res);
+    },
+    [folder, items, updateFolder],
+  );
+
+  /** function that will remove items from the folder when the button is clicked */
+  const removeItems = React.useCallback(async () => {
+    const updatedList = items.filter((item: any) => !selected.includes(item));
+    setItems(updatedList);
+    let res: IFolderModel | undefined;
+    if (!!folder) {
+      res = await updateFolder({
```

Comment:
You need to include `try+catch` just like you need a `.catch` for promises.  If you open your developer tools and execute this code it should give you a warning (normally in red) saying an unhandled promise.